### ISSUE TITLE
Autosave not working with zsh

### ISSUE THREAD

**Issue by scflode** [tmux-plugins/tmux-continuum#4]
Is there a known issue as of why the autosave mode might not work with zsh? The auto restore f.e. does work flawlessly though. Maybe I just miss a setting. Thanks for any hints! Attached my `.tmux.conf`.

```
# Copy-paste integration
set-option -g default-command "reattach-to-user-namespace -l zsh"

# Set default terminal color
set -g default-terminal "screen-256color"

# Use vim keybindings in copy mode
setw -g mode-keys vi
set -g status-keys vi

bind-key -t vi-edit Up   history-up
bind-key -t vi-edit Down history-down

# Setup 'v' to begin selection as in Vim
bind-key -t vi-copy v begin-selection
bind-key -t vi-copy y copy-pipe "reattach-to-user-namespace pbcopy"

# Update default binding of `Enter` to also use copy-pipe
unbind -t vi-copy Enter
bind-key -t vi-copy Enter copy-pipe "reattach-to-user-namespace pbcopy"

# Bind '<' to use pbpaste
bind < run "reattach-to-user-namespace pbpaste | tmux load-buffer - && tmux paste-buffer"

# Pane handling keybindings
bind-key j command-prompt -p "join pane from:"  "join-pane -s '%%'"
bind-key s command-prompt -p "send pane to:"  "join-pane -t '%%'"

# Clear screen
bind-key -n C-l send-keys C-z \; send-keys " reset && fg > /dev/null" \; send-keys "Enter"

# Add mouse support
setw -g mode-mouse on
set-option -g mouse-resize-pane on
set-option -g mouse-select-pane on
set-option -g mouse-select-window on

# Set window and pane index
set -g base-index 1
set -g pane-base-index 1

# continuum settings
set -g @continuum-restore 'on'
set -g @continuum-save-interval '1'

# tmux Plugins

set -g @tpm_plugins '          \
  tmux-plugins/tpm             \
  tmux-plugins/tmux-resurrect  \
  tmux-plugins/tmux-continuum  \
  tmux-plugins/tmux-yank       \
  tmux-plugins/tmux-copycat    \
' 

run-shell '~/.tmux/plugins/tpm/tpm'

# Load powerline
run-shell "powerline-daemon -q"
source "/Library/Python/2.7/site-packages/powerline/bindings/tmux/powerline.conf"
```
BTW thanks for this nice helpful plugin.

**Comment by bruno-** [comment]
Hey,
this same issue is reported in #2 (auto-save NOT working, auto-restore working ok).
As long as you have bash installed on the system things should work (no matter the user default shell is zsh).

I also experienced some issues with auto-save a few days back. That should be addressed with d6ac017d1dc7546ede2f66b2fb8015c720dce96a.

When did you install or last updated this plugin? Since the above bugfix was added ~3 days ago, maybe try pulling the latest version and see if things work?
